### PR TITLE
Update dice capsule with new Marketplace requirements

### PR DESCRIPTION
**Pull request by jatinbviv**

### CODE REVIEW

File: resources/en-US/capsule-info.bxb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+capsule-info {
+  dispatch-name (dice)
+  search-keywords{
+    keyword (dice)
+    keyword (die)
+    keyword (game)
+  }  
+  display-name (Dice Capsule)
+  developer-name (Bixby)
+  image-asset (/images/dice-icon.png)
```
Comment (sarahmolinerroy) [inline resources/en-US/capsule-info.bxb]: I would delete image-asset. It's not in use at the moment (marked as beta in the docs) and historically was a bigger different asset from the icon

File: resources/en-US/endpoints.bxb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+endpoints {
```
Comment (sarahmolinerroy) [inline resources/en-US/endpoints.bxb]: Did you mean to add this file as part of this?

Comment (jatinbviv) [reply resources/en-US/endpoints.bxb]: Actually, I had mistakenly created two locale folders, so I've removed one. Thanks for catching that.

File: resources/en-US/capsule-info.bxb (hunk 1 above)
Comment (jatinbviv) [reply resources/en-US/capsule-info.bxb]: Thanks. Done. 👍

File: resources/en/capsule-info.bxb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+capsule-info {
+  dispatch-name (dice)
+  search-keywords{
+    keyword (dice)
+    keyword (die)
```
Comment (sarahmolinerroy) [inline resources/en/capsule-info.bxb:5]: I know it's the plural, but it always looks weird standalone...

**Review by sarahmolinerroy — APPROVED** [review]

**Comment by jatinbviv** [comment]
Closing my PR since Paulina has a newer one: https://github.com/bixbydevelopers/capsule-sample-dice/pull/10

**Pull request closed without merging**